homeassistant/components/harmony/connection_state.py

```diff
@@ -0,0 +1,43 @@
+"""Mixin class for handling connection state changes."""
+import logging
+
+from homeassistant.helpers.event import async_call_later
+
+_LOGGER = logging.getLogger(__name__)
+
+TIME_MARK_DISCONNECTED = 10
+
+
+class ConnectionStateMixin:
+    """Base implementation for connection state handling."""
+
+    def __init__(self):
+        """Initialize this mixin instance."""
+        self._unsub_mark_disconnected = None
```

Comment:
```suggestion
        super().__init__()
        self._unsub_mark_disconnected = None
```

This may need a super init.  Need to check

Reply:
Good catch - it seemed to work without but best to have it be consistent.